project_cost_report/controller/print_report.py

```diff
@@ -0,0 +1,30 @@
+# © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import json
+from odoo import http
+from odoo.http import request
+
+
+class PrintProjectCostReport(http.Controller):
+
+    @http.route('/web/project_cost_report/<int:project_id>', type='http', auth='user')
+    def project_cost_report_pdf(self, project_id, report_context, token):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=3675949)